src/containers/pages/FocusInvestigation/map/active/tests/indexCases.test.tsx

```diff
@@ -153,4 +156,52 @@ describe('containers/pages/FocusInvestigation/activeMap', () => {
       expect(localLayer).toMatchSnapshot();
     });
   });
+
+  it('displays map when no structures are returned', async () => {
+    // use dispatch
+    store.dispatch(fetchPlans(fixturesMap.processedPlansJSON));
+    store.dispatch(fetchJurisdictions(fixturesMap.processedJurisdictionJSON));
+    store.dispatch(structureDucks.setStructures([]));
+    store.dispatch(fetchGoals(fixturesMap.processedGoalsJSON));
+    store.dispatch(fetchTasks(fixturesMap.processedPlansTasksJson));
+    store.dispatch(fetchTasks(fixturesMap.processedCaseConfirmationTasksJSON));
+    const mock = jest.fn();
+    const supersetServiceMock = jest.fn(async () => null);
+    const props = {
+      history,
+      location: mock,
+      match: {
+        isExact: true,
+        params: { id: 'dbd9851f-2548-5aaa-8267-010897f98f45' },
+        path: `${FI_SINGLE_URL}/:id`,
+        url: `${FI_SINGLE_URL}/dbd9851f-2548-5aaa-8267-010897f98f45`,
+      },
+      supersetService: supersetServiceMock,
+    };
+    const wrapper = mount(
+      <Provider store={store}>
+        <Router history={history}>
+          <ConnectedMapSingleFI {...props} />
+        </Router>
+      </Provider>
+    );
+
+    await new Promise(resolve => setImmediate(resolve));
+    wrapper.update();
+
+    const componentProps: any = wrapper.find('SingleActiveFIMap').props();
+
+    expect(componentProps.plan.id).toEqual('dbd9851f-2548-5aaa-8267-010897f98f45');
+    expect(componentProps.jurisdiction.jurisdiction_id).toEqual('3951');
+    expect(componentProps.pointFeatureCollection.features.length).toEqual(95);
+    expect(componentProps.polygonFeatureCollection.features.length).toEqual(7);
+    expect(componentProps.currentPointIndexCases.features.length).toEqual(1);
+    expect(componentProps.currentPolyIndexCases.features.length).toEqual(0);
+    expect(componentProps.historicalPointIndexCases.features.length).toEqual(0);
+    expect(componentProps.historicalPolyIndexCases.features.length).toEqual(2);
```

Comment:
Lets add a test where all these are empty i.e. everything apart from the jurisdiction boundary is absent